eblocker-ui/src/settings/app/components/openSourceLicenses/raw/javascript-dependencies.json

```diff
@@ -151,7 +151,6 @@
 {"name":"from2","version":"2.3.0","repository":"http://github.com/hughsk/from2","summary":"MIT"},
 {"name":"fs-write-stream-atomic","version":"1.0.10","repository":"https://github.com/npm/fs-write-stream-atomic","summary":"ISC"},
 {"name":"fs.realpath","version":"1.0.0","repository":"https://github.com/isaacs/fs.realpath","summary":"ISC"},
-{"name":"fsevents","version":"2.1.3","repository":"(none)","summary":"MIT"},
```

Comment:
This change was made automatically by the maven build. Do these need to be checked in?